macros/cssxref.ejs

```diff
@@ -73,7 +68,7 @@ if (!$2) {
     //if (!thisPage) thisPage = wiki.getPage(url); // lazy evaluation // Disabled to test performance (we get a lot of Kumascript errors)
 
     if (thisPage && thisPage.summary) {
-        summary = mdn.escapeQuotes(thisPage.summary);
+        summary = kuma.htmlEscape(thisPage.summary);
```

Comment:
I'm not sure this is what we want here. It will produce tooltips with HTML as text:

"The \<strong>\<code>\&lt;length\&gt;\</code>\</strong> CSS data type represents a distance value."